tutormfe/templates/mfe/build/mfe/Dockerfile

```diff
@@ -53,6 +53,7 @@ COPY --from={{ app["name"] }}-src /openedx/app /openedx/app
 COPY --from={{ app["name"] }}-i18n /openedx/app/src/i18n/messages /openedx/app/src/i18n/messages
 ENV PUBLIC_PATH='/{{ app["name"] }}/'
```

Comment:
We should remove `PUBLIC_PATH` here and set it in the build line below.

Reply:
Because It's needed only for production I guess, given that on development each MFE has its own domain/port?